.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+_Add ticket reference to Pull Request title: e.g. 'FS-123: Add content', if there is no ticket prefix with BAU_
+
+
+### Change description
+_A brief description of the pull request_ 
+
+- [ ] Unit tests and other appropriate tests added or updated
+- [ ] README and other documentation has been updated / added (if needed)
+- [ ] JIRA ticket updated accordingly
+- [ ] Commit messages are meaningful and follow good commit message guidelines
+
+
+### How to test
+_If manual testing is needed, give suggested testing steps_
+
+
+### How should it be reviewed? 
```

Comment:
How would this section be used? should the change requester have a say on how their PR should be reviewed (conflict of interest and all that?)

Reply:
@gidsg  your thoughts on Gio's point?